frontend/homework/10/todo-list/src/styles/ItemManagerStyles/Item.scss

```diff
@@ -0,0 +1,28 @@
+* {
```

Comment:
All the colors, font related values must be stored in the different constants file